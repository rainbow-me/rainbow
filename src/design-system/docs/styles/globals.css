@font-face {
  font-family: 'SFRounded';
  src: url('/fonts/subset-SFRounded-Regular.eot');
  src: url('/fonts/subset-SFRounded-Regular.eot?#iefix')
      format('embedded-opentype'),
    url('/fonts/subset-SFRounded-Regular.woff2') format('woff2'),
    url('/fonts/subset-SFRounded-Regular.woff') format('woff'),
    url('/fonts/subset-SFRounded-Regular.svg#SFRounded-Regular') format('svg');
  font-weight: 400;
  font-style: normal;
  font-display: auto;
}

@font-face {
  font-family: 'SFRounded';
  src: url('/fonts/subset-SFRounded-Medium.eot');
  src: url('/fonts/subset-SFRounded-Medium.eot?#iefix')
      format('embedded-opentype'),
    url('/fonts/subset-SFRounded-Medium.woff2') format('woff2'),
    url('/fonts/subset-SFRounded-Medium.woff') format('woff'),
    url('/fonts/subset-SFRounded-Medium.svg#SFRounded-Medium') format('svg');
  font-weight: 500;
  font-style: normal;
  font-display: auto;
}

@font-face {
  font-family: 'SFRounded';
  src: url('/fonts/subset-SFRounded-Semibold.eot');
  src: url('/fonts/subset-SFRounded-Semibold.eot?#iefix')
      format('embedded-opentype'),
    url('/fonts/subset-SFRounded-Semibold.woff2') format('woff2'),
    url('/fonts/subset-SFRounded-Semibold.woff') format('woff'),
    url('/fonts/subset-SFRounded-Semibold.svg#SFRounded-Semibold') format('svg');
  font-weight: 600;
  font-style: normal;
  font-display: auto;
}

@font-face {
  font-family: 'SFRounded';
  src: url('/fonts/subset-SFRounded-Bold.eot');
  src: url('/fonts/subset-SFRounded-Bold.eot?#iefix')
      format('embedded-opentype'),
    url('/fonts/subset-SFRounded-Bold.woff2') format('woff2'),
    url('/fonts/subset-SFRounded-Bold.woff') format('woff'),
    url('/fonts/subset-SFRounded-Bold.svg#SFRounded-Bold') format('svg');
  font-weight: 700;
  font-style: normal;
  font-display: auto;
}

@font-face {
  font-family: 'SFRounded';
  src: url('/fonts/subset-SFRounded-Heavy.eot');
  src: url('/fonts/subset-SFRounded-Heavy.eot?#iefix')
      format('embedded-opentype'),
    url('/fonts/subset-SFRounded-Heavy.woff2') format('woff2'),
    url('/fonts/subset-SFRounded-Heavy.woff') format('woff'),
    url('/fonts/subset-SFRounded-Heavy.svg#SFRounded-Heavy') format('svg');
  font-weight: 800;
  font-style: normal;
  font-display: auto;
}

html,
body {
  padding: 0;
  margin: 0;
  font-family: 'SFRounded', ui-rounded, 'SF Pro Rounded', system-ui,
    'Helvetica Neue', Arial, Helvetica, sans-serif;
  text-rendering: optimizeLegibility;
  background-color: #e9f2ff;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
  -webkit-text-size-adjust: 100%;
}

* {
  box-sizing: border-box;
  margin: 0;
}
