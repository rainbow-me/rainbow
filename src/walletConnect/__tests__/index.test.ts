import { jest, test } from '@jest/globals';

jest.mock('@walletconnect/core');

jest.mock('@/redux/store');
jest.mock('@/redux/walletconnect');
jest.mock('@/redux/requests');
jest.mock('@/navigation/Navigation');
jest.mock('@/handlers/imgix');
jest.mock('@/utils/ethereumUtils');
jest.mock('@/parsers/requests');
jest.mock('@/handlers/localstorage/walletconnectRequests');
jest.mock('@/handlers/appEvents');
jest.mock('@/notifications/tokens');
jest.mock('@/model/wallet');
jest.mock('@/handlers/web3');
jest.mock('@/screens/Portal');
jest.mock('@/walletConnect/sheets/AuthRequest', () => ({
  AuthRequest: jest.fn(),
}));

// import { parseRPCParams } from '@/walletConnect';
// import { RPCMethod } from '@/walletConnect/types';

test(`works`, () => {});
