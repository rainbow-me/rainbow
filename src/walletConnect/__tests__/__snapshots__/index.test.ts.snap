// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parseRPCParams 1`] = `
Object {
  "address": "0xA2Eaa7BAe79F0F9FfB23667cdAc9CE285b30aE0E",
}
`;

exports[`parseRPCParams 2`] = `
Object {
  "address": "0xA2Eaa7BAe79F0F9FfB23667cdAc9CE285b30aE0E",
  "message": "My email is john@doe.com - 1683582604485",
}
`;

exports[`parseRPCParams 3`] = `
Object {
  "address": "0xA2Eaa7BAe79F0F9FfB23667cdAc9CE285b30aE0E",
  "message": "My email is john@doe.com - 1683582643859",
}
`;

exports[`parseRPCParams 4`] = `
Object {
  "address": "0xA2Eaa7BAe79F0F9FfB23667cdAc9CE285b30aE0E",
  "message": Object {
    "domain": Object {
      "chainId": 1,
      "name": "Ether Mail",
      "verifyingContract": "0xCcCCccccCCCCcCCCCCCcCcCccCcCCCcCcccccccC",
      "version": "1",
    },
    "message": Object {
      "contents": "Hello, Bob!",
      "from": Object {
        "name": "Cow",
        "wallet": "0xCD2a3d9F938E13CD947Ec05AbC7FE734Df8DD826",
      },
      "to": Object {
        "name": "Bob",
        "wallet": "0xbBbBBBBbbBBBbbbBbbBbbbbBBbBbbbbBbBbbBBbB",
      },
    },
    "primaryType": "Mail",
    "types": Object {
      "EIP712Domain": Array [
        Object {
          "name": "name",
          "type": "string",
        },
        Object {
          "name": "version",
          "type": "string",
        },
        Object {
          "name": "chainId",
          "type": "uint256",
        },
        Object {
          "name": "verifyingContract",
          "type": "address",
        },
      ],
      "Mail": Array [
        Object {
          "name": "from",
          "type": "Person",
        },
        Object {
          "name": "to",
          "type": "Person",
        },
        Object {
          "name": "contents",
          "type": "string",
        },
      ],
      "Person": Array [
        Object {
          "name": "name",
          "type": "string",
        },
        Object {
          "name": "wallet",
          "type": "address",
        },
      ],
    },
  },
}
`;
